src/ulohy/brainfuck.md

```diff
@@ -0,0 +1,49 @@
+# Překladač jazyka Brainfuck
+
+Jazyk [Brainfuck](https://esolangs.org/wiki/Brainfuck) je velmi jednoduchý esoterický programovací
+jazyk obsahující pouze osm operátorů, lineární paměť a adresu aktuální paměťové buňky. Jedna buňka
+odpovídá jednomu bytu.
+
+* Operátor `+` inkrementuje hodnotu aktuální buňky.
+* Operátor `-` dekrementuje hodnotu aktuální buňky.
+* Operátor `>` inkrementuje adresu buňky (po inkrementaci ukazujeme na následující buňku).
+* Operátor `<` dekrementuje adresu buňky (po dekrementaci ukazujeme na předcházející buňku).
+* Operátor `[` uvozuje začátek cyklu. Cyklus probíhá, dokud hodnota buňky adresované v době
+  vyhodnocování podmínky není nulová (adresovanou buňku můžeme měnit uvnitř cyklu).
+* Operátor `]` představuje konec těla cyklu, ale nereprezentuje žádné chování.
```

Comment:
This is a bit confusing I think, when you encounter `]`, there is behavior, as described below.

Reply:
In a proper implementation (i.e. one which creates an AST), the `[` and `]` operators would indeed carry no meaning beyond syntactic denotation of a loop, which would still behave as described by the language spec. However, since we probably want the students to start with an implementation which interprets the string directly, I agree it's better to reword the description.